packages/flutter/test/widgets/test_widgets.dart

```diff
@@ -56,3 +57,49 @@ class FlipWidgetState extends State<FlipWidget> {
 void flipStatefulWidget(WidgetTester tester) {
   tester.state<FlipWidgetState>(find.byType(FlipWidget)).flip();
 }
+
+class TestScrollable extends StatelessWidget {
```

Comment:
I'm slightly suspicious of wanting this in tests but not in real apps, but I guess we'll see.